user_manual/talk/advanced_features.rst

```diff
@@ -70,3 +70,46 @@ From within Deck, you can share tasks into chat conversations.
 
 .. image:: images/deck-talk-share-card-to-chat-in-talk.png
 
+Breakout rooms
+--------------
+Breakout rooms allow you to divide a Nextcloud Talk call into smaller groups for more focused discussions. The moderator
+of the call can create multiple breakout rooms and assign participants to each room.
+
+Configure breakout rooms
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+To create breakout rooms, you need to be a moderator in a group conversation. Click on the top-bar menu and click on
+"Setup breakout rooms".
+
+.. image:: images/talk-breakout-rooms-setup.png
+
+A dialog will open where you can specify the number of rooms you want to create and the participants assignment method.
+Here you'll be presented with 3 options:
+- **Automatically assign participants**: Talk will automatically assign participants to the rooms.
+- **Manually assign participants**: You'll go through a participants editor where you can assign participants to rooms.
+- **Allow participants choose**: Participants will be able to join breakout rooms themselves.
+
+.. image:: images/talk-breakout-rooms-setup-dialog.png
+
+Manage breakout rooms
+^^^^^^^^^^^^^^^^^^^^^
+
+Once the breakout rooms are created, you will be able to see them in the sidebar.
+
+.. image:: images/talk-breakout-rooms-sidebar.png
+
+From the sidebar header
+
+- **Start and stop the breakout rooms**: this will move all the users in the parent conversation to their respective
+breakout rooms.
+- **Broadcast a message to all the rooms**: this will send a message to all the rooms at the same time.
+- **Make changes to the assigned participants**: this will open the participants editor where you can change which
+participans are assigned to which breakout room. From this dialog it's also possible to delete the breakout rooms.
```

Comment:
Bullet list ends without a blank line; unexpected unindent.

```suggestion
- **Make changes to the assigned participants**: this will open the participants editor where you can change which participans are assigned to which breakout room. From this dialog it's also possible to delete the breakout rooms.
```